package.json

```diff
@@ -55,17 +55,24 @@
     "typescript": "^5.1.6"
   },
   "peerDependencies": {
-    "@langchain/core": "^0.1.23"
+    "langchain": "^0.1.19",
+    "@langchain/core": "^0.1.19"
   },
   "peerDependenciesMeta": {
+    "langchain": {
+      "optional": false
```

Comment:
@petergarbers Does this mean any install of `zep-js` will install `langchain`? Ideally this is only needed if the LangChain classes are used

Reply:
@danielchalef no it won't. It's just to show a warning during installation. 
![CleanShot 2024-02-19 at 14 03 50@2x](https://github.com/getzep/zep-js/assets/26054637/534ad59b-d196-4589-b1fe-93af59c12596)

Reply:
If we don't set option to false, no warning will be shown to the user when installing zep-js without langchain present.

Reply:
OK - we discussed making the peerDep optional. Happy to resolve this.